content-src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -104,11 +102,22 @@ ActivityFeedItem.propTypes = {
   type: React.PropTypes.string,
   dateDisplay: React.PropTypes.number,
   provider_display: React.PropTypes.string,
-  parsedUrl: React.PropTypes.shape({
-    hostname: React.PropTypes.string
-  })
+  parsedUrl: React.PropTypes.shape({hostname: React.PropTypes.string})
 };
 
+function groupSitesBySession(sites) {
+  const sessions = [[]];
+  sites.forEach((site, i) => {
+    const currentSession = sessions[sessions.length - 1];
+    const nextSite = sites[i + 1];
+    currentSession.push(site);
+    if (nextSite && Math.abs(site.dateDisplay - nextSite.dateDisplay) > SESSION_DIFF) {
+      sessions.push([]);
+    }
+  });
+  return sessions;
+}
```

Comment:
Ah, I think I always specified `"nofunc"` so it'd ignore hoisted functions-n-stuff.